crates/rpc/rpc-builder/src/lib.rs

```diff
@@ -1230,6 +1230,8 @@ pub struct RpcServerConfig {
     ipc_endpoint: Option<Endpoint>,
     /// JWT secret for authentication
     jwt_secret: Option<JwtSecret>,
+    /// HTTP RPC path prefix.
+    http_rpcprefix: Option<String>,
```

Comment:
okay now we need to figure out how to actually use this.

I think we need to solve this via a Middleware that checks the Uri?
maybe something like this exists already?

Reply:
> ybe something like this exists already?

i'm not sure i found something like this, do you have any suggestion please ?

Reply:
hmm, I don't think this exists, so we'd need to add this as a layer instead

Reply:
> we'd need to add this as a layer instead that checks the request's uri

you means i need to implement Layer and Service for type that contain prefix field ?